specification/metrics/pricingquantity.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Quantity
+
+The Pricing Quantity represents the volume of a given resource and/or service used or purchased based on the [Pricing Unit](#pricingunit). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not resource and service consumption.
+
+PricingQuantity MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [ChargeType](#chargetype) is "Adjustment". This column MUST NOT contain null values where unit prices are not null. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Comment:
```suggestion
PricingQuantity MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [ChargeCategory](#chargecategory) is "Adjustment". This column MUST NOT contain null values where unit prices are not null. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Reply:
Do we need to be explicit about ChargeSubcategory: Credit? Should we add something like this: 
`...and MUST be negative in cases where [ChargeSubcategory](#chargesubcategory) is 'Credit'`